db/migrate/20241128141813_update_withdrawal_reasons.rb

```diff
@@ -5,7 +5,5 @@ def up
     WithdrawalReason.upsert_all(WithdrawalReasons::SEED, unique_by: :name)
   end
 
-  def down
-    raise ActiveRecord::IrreversibleMigration
-  end
+  def down; end
```

Comment:
this migration is idempotent, so I've removed the error raised here (you'll need to re-run to get that extra reason mentioned above in)